src/js/legend_config.js

```diff
@@ -35,6 +36,10 @@ export const sections = [
     name: "Route markers",
     source: "Wikidata",
   },
+  {
+    name: "Points of Interest",
```

Comment:
Should we move this section below Aviation to keep the transportation-related sections together?

Reply:
Resolved in cf9dbf1.